addons/website/static/src/components/wysiwyg_adapter/wysiwyg_adapter.js

```diff
@@ -361,7 +361,8 @@ export class WysiwygAdapterComponent extends Wysiwyg {
      * @returns {HTMLElement}
      */
     get editable() {
-        return this.websiteService.pageDocument.getElementById('wrapwrap');
+        // Page document might become unavailable when leaving the page.
+        return this.websiteService.pageDocument?.getElementById('wrapwrap');
```

Comment:
Needed because this.$editable[0] call the getter, which crashed to begin with.

Since you also change the other line, do you think it would be a good idea to directly use `this.editable` instead of `this.$editable[0]`?